### PR TITLE
O3-3765: (Test) Add Tests for stock source delete

### PR DESCRIPTION
**Pull request by ODORA0**
## Requirements
- [ ] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Related Issue
https://openmrs.atlassian.net/browse/O3-3765

### CODE REVIEW

**Comment by chimanwadike** [comment]
Well done @ODORA0 
Can you update the PR title to include the JIRA issue/ticket number and also add the link to the related issue in the PR summary, please?

Same applies to some of your open PRs.

Thanks a lot

File: src/stock-sources/stock-sources-delete/stock-sources-delete.test.tsx [hunk 1]
```diff
@@ -0,0 +1,190 @@
+import React from "react";
+import { render, fireEvent, waitFor, screen } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import { showModal, showSnackbar } from "@openmrs/esm-framework";
+import { deleteStockSource } from "../stock-sources.resource";
+import StockSourcesDeleteActionMenu from "./stock-sources-delete.component";
```
Comment (jabahum) [inline src/stock-sources/stock-sources-delete/stock-sources-delete.test.tsx:6]: we can remove unused imports

Comment (ODORA0) [reply src/stock-sources/stock-sources-delete/stock-sources-delete.test.tsx:6]: @jabahum This import is being used, which one in specific did you mean

**Review by jabahum — COMMENTED** [review]
Thanks @ODORA0

**Pull request merged**